source/includes/_investor-payments.md

```diff
@@ -567,3 +567,43 @@ Please note this is a test endpoint and is only available in the sandbox environ
 | paymentReference | string | The reference. Set to ISA if the funds should be credited to the ISA account. | required |
 | paymentType      | string | Should be set to DEPOSIT                                                      | required |
 | clientId         | string | The client ID                                                                 | required |
+
+## `POST /platformApi/user/{userId}/fee`
+
+```http
+
+POST /platformApi/user/{userId}/fee HTTP/1.1
```

Comment:
docs look ok... this comment is less in relation to your docs and more to do with the endpoint URI... wondering if it's a bit weird that everything else under the 'Payments - Investors' section have the format of /platformApi/investors/{clientId}/... whereas meanwhile this new endpoint goes: /platformApi/user/{userId}/fee and essentially references the internal investorId (userId) instead of the external clientId?

Reply:
I agree with Vic. User Id isn't exposed to platforms so this should be clientId. Platforms can only reference investors using clientId. We likely will need a better way of referencing entities as not all of them will be investors going forward but way out of scope for this :)

This probably means the change in Payments needs to lookup the user based on the clientId.